README.md

```diff
@@ -85,12 +86,25 @@ subscribeEvent_ :: forall m a. MonadEffect m => MonadCleanup m => (a -> Effect U
 #### [Callback](https://pursuit.purescript.org/packages/purescript-specular/docs/Specular.Callback#t:Callback)
 
 `Callback a` represents an action handler which consumes a value of type `a`. Think of it as `a -> Effect Unit`.
+This represenation is likely to change in the future to `a -> Effect Unit`.
 
 ```purescript
 -- | Trigger the action in Effect.
 triggerCallback :: forall a. Callback a -> a -> Effect Unit
 ```
 
+The DOM API that Specular exposes accepts callbacks, please also have a look at `Ref` bellow.
+
+```purescript
+-- | Create a button that triggers `removeTask` callback
+el "button" [class_ "close", attrs ("type":="button" ), onClick_ $ removeTask task ] do
+  text "Remove"
+
+-- | Define a callback which modifies the content of `tasks` `Ref` by filtering out a task with a given id
+let removeTask task = mkCallback \_ ->  triggerCallback (Ref.modify tasks) (filter $ \c -> c.id /= task.id)
```

Comment:
While we're still using `Callback`, this is more idiomatic:

```suggestion
let removeTask task = cmap (\_ ->  filter \c -> c.id /= task.id) (Ref.modify tasks)
```

Reply:
I wanted to provide both examples, I think it did in the `Counter` example, but maybe I didn't here.